tools/topology/sof-bdw-codec.m4

```diff
@@ -17,15 +17,14 @@ include(`sof/tokens.m4')
 # Include Broadwell DSP configuration
 include(`platform/intel/bdw.m4')
 
-
 define(PIPE_NAME, pipe-bdw-`'CODEC`')
 
 #
 # Define the pipelines
 #
-# PCM0 ----> volume ---------------+
-#                                  |--low latency mixer ----> volume ---->  SSP0
-# PCM1 -----> volume -----> SRC ---+
+# PCM0 ----> volume ------+
```

Comment:
We are good on BDW since it has way more memory that BYT.

Reply:
> Do we still need mixer on BDW after we remove SRC

I think we need test cases for mixer.

@lgirdwood OK, I'll leave BDW with SRC. And keep better SRC in APL.

Reply:
Note to myself: This update is not yet done. I can't reject my own PR.

Reply:
> I can't reject my own PR.

I know, this sucks! On the other hand you can "downgrade" it to a draft. This is better in some way because it removes it from the radar of overbusy and smart maintainers who don't look at drafts.